details/CLUSTER_DEFINITION.md

```diff
@@ -62,15 +64,17 @@ This example here shows a cluster definition in JSON format, as it may be submit
     "workers": [
         {
             "aws": {
-              "instance_type": "m3.large"
+                "instance_type": "m3.large"
```

Comment:
missing a comma at the end of the line here.

Reply:
same here

Reply:
talking about: https://github.com/giantswarm/api-spec/pull/108/files#r214950270